compute/src/test/java/org/jclouds/compute/internal/BaseComputeServiceLiveTest.java

```diff
@@ -386,11 +386,11 @@ public void testCreateTwoNodesWithOneSpecifiedName() throws Exception {
       this.nodes = newTreeSet(concat(this.nodes, nodes));
    }
 
-   private Template refreshTemplate() {
+   protected Template refreshTemplate() {
       return template = addRunScriptToTemplate(buildTemplate(client.templateBuilder()));
    }
 
-   protected static Template addRunScriptToTemplate(Template template) {
+   protected Template addRunScriptToTemplate(Template template) {
       template.getOptions().runScript(Statements.newStatementList(AdminAccess.standard(), InstallJDK.fromOpenJDK()));
```

Comment:
If subclasses need to override this, does it mean `AdminAccess.standard()` doesn't work there? That might be an issue to address too?